run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/zsh
```

Comment:
maybe better if it was '#!/bin/bash' since zsh does not come pre-packaged with every distro